Assets/ZSSRichTextEditor.js

```diff
@@ -1991,6 +2028,27 @@ ZSSField.prototype.bindListeners = function() {
     this.wrappedObject.bind('input', function(e) { thisObj.handleInputEvent(e); });
     this.wrappedObject.bind('compositionstart', function(e) { thisObj.handleCompositionStartEvent(e); });
     this.wrappedObject.bind('compositionend', function(e) { thisObj.handleCompositionEndEvent(e); });
+    
+    var target = this.wrappedObject[0];
```

Comment:
Just to separate concerns a bit, I'd recommend to place the code to create / initialize the mutation observer in a method of it's own and call it here with a single line instead.  It feels like this method should handle a higher level logic.